troubleshooting/known_issues/missing_binaries_in_aarch64_images.md

```diff
@@ -0,0 +1,120 @@
+# Missing binaries in aarch64 Images
+
+Due to a bug in the QEMU emulation software, there are times that dependencies are missed
```

Comment:
I don't know why, but the HPE style guide only wants the phrase "due to" used if the surrounding context is "X is due to Y". Otherwise, it says to use "because of". 
```suggestion
Because of a bug in the QEMU emulation software, there are times that dependencies are missed
```